graphene_django/templates/graphene/graphiql.html

```diff
@@ -45,6 +45,9 @@
     {% if subscription_path %}
       subscriptionPath: "{{subscription_path}}",
     {% endif %}
+    {% if header_editor_enabled %}
+      graphiqlHeaderEditorEnabled: "{{ graphiql_header_editor_enabled}}",
```

Comment:
doesn't that mean it will be a `"true"` or `"false"` (string instead of boolean)?